### PR TITLE
Allow specification of user_verification level

### PR DESCRIPTION
**Pull request by MathisRosenhauer**
Chrome always asks me for the PIV PIN of my YubiKey 5 when authenticating or registering. This allows to set `user_verification=UserVerificationRequirement.DISCOURAGED` to authenticate by touch only.

### CODE REVIEW

**Comment by MathisRosenhauer** [comment]
Let me fix the problems with older fido2

File: django_fido/views.py [hunk 1]
```diff
@@ -49,11 +53,13 @@ class Fido2ViewMixin(object):
                        see https://www.w3.org/TR/webauthn/#enumdef-attestationconveyancepreference
     @cvar attestation_types: Allowed attestation format types.
         If None, all attestation formats except `none` are allowed.
+    @cvar user_verification: The desired USER_VERIFICATION level.
```
Comment (ziima) [inline django_fido/views.py]: I'd use `Requirements of user verification` to match the terms used in specs.

File: django_fido/views.py [hunk 2]
```diff
@@ -49,11 +53,13 @@ class Fido2ViewMixin(object):
...
     @cvar session_key: Session key where the FIDO 2 state is stored.
     """
 
     attestation = AttestationConveyancePreference.NONE
     attestation_types = None  # type: Optional[List[Attestation]]
+    user_verification = UserVerificationRequirement.PREFERRED if fido2.__version__ < '0.8' else None
```
Comment (ziima) [inline django_fido/views.py]: `PREFERRED` should work in older fido2 versions as well, shouldn't it?

File: django_fido/views.py (hunk 1 above)
Comment (MathisRosenhauer) [reply django_fido/views.py]: I copied that from the fido2 package but the specs are certainly better.

File: django_fido/views.py (hunk 2 above)
Comment (MathisRosenhauer) [reply django_fido/views.py]: I thought maybe not to prescribe the default. It may be cleaner to set it, though. Tests have to be adapted in that case.

Comment (ziima) [reply django_fido/views.py]: I've checked the wrong version of `fido2` library. Sorry.

File: django_fido/views.py (hunk 1 above)
Comment (ziima) [reply django_fido/views.py]: Sadly, the `fido2` descriptions are not very descriptive.

File: django_fido/views.py [hunk 3]
```diff
@@ -53,7 +53,8 @@ class Fido2ViewMixin(object):
                        see https://www.w3.org/TR/webauthn/#enumdef-attestationconveyancepreference
     @cvar attestation_types: Allowed attestation format types.
         If None, all attestation formats except `none` are allowed.
-    @cvar user_verification: The desired USER_VERIFICATION level.
+    @cvar user_verification: Requirement of user verification,
+        see https://www.w3.org/TR/webauthn/#userVerificationRequirement
```
Comment (MathisRosenhauer) [inline django_fido/views.py]: Would this be sufficient?

Comment (ziima) [reply django_fido/views.py]: Yes.

**Comment by MathisRosenhauer** [comment]
Squashed the commits.

**Pull request merged**